addons/web/static/tests/fields/field_utils_tests.js

```diff
@@ -148,6 +148,28 @@ QUnit.test('parse float', function(assert) {
     core._t.database.parameters = originalParameters;
 });
 
+QUnit.test('parse integer', function(assert) {
+    assert.expect(8);
+
+    assert.strictEqual(fieldUtils.parse.integer(""), 0);
+    assert.strictEqual(fieldUtils.parse.integer("0"), 0);
+    assert.strictEqual(fieldUtils.parse.integer("100"), 100);
+    assert.strictEqual(fieldUtils.parse.integer("-100"), -100);
+    assert.strictEqual(fieldUtils.parse.integer("1,000"), 1000);
+    assert.strictEqual(fieldUtils.parse.integer("1,000,000"), 1000000);
+    assert.strictEqual(fieldUtils.parse.integer("1,000,000.00"), 1000000);
```

Comment:
In order for the test to be more relevant, it may be necessary to use a non-zero value for decimals

Reply:
@Gorash fixed!